.github/workflows/lint.yml

```diff
@@ -30,7 +30,6 @@ jobs:
       - name: Run Forge build
         run: |
           forge --version
-          forge build --sizes
```

Comment:
looks like we've hit the size limit for QuarkBuilder.sol; CI fails this check currently